src/Client.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef Client_hpp
+#define Client_hpp
+
+#include <SFML/Graphics.hpp>
+#include "Group.hpp"
+#include "GroupShape.hpp"
+#include <stdio.h>
+
+class Client {
+
+    public:
+        Client(int player_count);
+        ~Client();
+
+        void draw(sf::RenderTarget& target);
+        void update(std::vector<Group*> groups);
+
+    private:
+        std::vector<GroupShape*> mGroupShapes;
```

Comment:
The state in this class is a bit weird to me. Why does it keep group shapes but not groups? What is client meant to represent?

Reply:
Discussed offline